kt/godot-library/src/main/kotlin/godot/global/GDRandom.kt

```diff
@@ -8,20 +8,15 @@ internal interface GDRandom {
     var rng: RandomNumberGenerator?
 
     /** Random range, any floating point value between from and to. */
-    fun randRange(from: Float, to: Float) = randRange(from.toDouble(), to.toDouble()).toFloat()
-
-    /** Random range, any floating point value between from and to. */
-    fun randRange(from: Double, to: Double) = rng!!.randfRange(from, to).toDouble()
+    fun randRange(from: Float, to: Float) = rng!!.randfRange(from, to).toDouble()
```

Comment:
I created 2 versions of those functions so people can freely use Float or Double for whatever is the most convenient at the moment. I don't think we should remove them. Kotlin's Math package is also full of doubles like this. 
We don't gain anything from removing more convenience for users.